MapboxCoreNavigation/RouteController.swift

```diff
@@ -330,6 +332,54 @@ extension RouteController: CLLocationManagerDelegate {
         }
     }
     
+    func checkForFasterRoute(from location: CLLocation) {
+        
+        // If the user is on a short route, don't check for faster alternatives
```

Comment:
I'm not certain I see why distance remaining would be more useful here. If there is not much distance remaining on a route, but the duration remaining is high, we should still be looking for a new route.